setup.py

```diff
@@ -20,8 +20,7 @@
 ]
 
 requires = [
-    'anyjson>=0.3.3',
-    'kombu>=3.0.37',
+    'kombu>=4.0.0',
     'cryptography>=2.0.2',
```

Comment:
Does this need to be bumped up since it's been pinned to a higher version in `requirements.txt`?

Reply:
Ok, thanks for clarifying that.